client/client.go

```diff
@@ -178,7 +179,15 @@ func (kc *kClient) initRPCInfo(ctx context.Context, method string) (context.Cont
 	rmt := remoteinfo.NewRemoteInfo(kc.opt.Svr, method)
 	ctx = kc.applyCallOptions(ctx, cfg.ImmutableView(), rmt)
 	rpcStats := rpcinfo.AsMutableRPCStats(rpcinfo.NewRPCStats())
-	rpcStats.SetLevel(kc.opt.StatsLevel)
+	if kc.opt.StatsLevel != nil {
+		rpcStats.SetLevel(*kc.opt.StatsLevel)
+	} else {
+		if kc.opt.TracerCtl.HasTracer() {
+			rpcStats.SetLevel(stats.LevelDetailed)
+		} else {
+			rpcStats.SetLevel(stats.LevelBase)
```

Comment:
Why don't set disabled when no Tracer?

Reply:
fix updated

Reply:
It is not suitable check here, initRPCInfo() is executed for every call